app/models/payment.rb

```diff
@@ -0,0 +1,64 @@
+class Payment < ActiveRecord::Base
+  has_many :ticket_purchases
+  belongs_to :user
+  belongs_to :conference
+
+  attr_accessor :credit_card_number
+  attr_accessor :credit_card_type
+  attr_accessor :card_verification_value
+  attr_accessor :expiration_month
+  attr_accessor :expiration_year
+
+  validates :first_name, presence: true
```

Comment:
Do we need to store all that data in our database? AFAIK we need to implement the [PCI](https://en.wikipedia.org/wiki/Payment_Card_Industry_Data_Security_Standard) standard for events.o.o ...